presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -184,6 +184,10 @@ Property Name                                      Description
 
 ``hive.s3select-pushdown.max-connections``         Maximum number of simultaneously open connections to S3 for  500
                                                    S3SelectPushdown.
+
+``hive.recursive-directories``                     Enable reading data from subdirectories of a specific path   ``false``
```

Comment:
Let's move this item above to be after the `hive.config.resources` description.

For the description, how about
```
Enable reading data from subdirectories of table or
partition locations. If disabled, subdirectories are
ignored. This is equivalent to the
``hive.mapred.supports.subdirectories`` property in Hive.